### DISCUSSION TITLE
Windows Installer Issue

### DISCUSSION THREAD

**Discussion by Starukami** [agersant/polaris#202] [Q&A] [answered]
Tried to install on windows 11 23H2, installer worked but when launching I get this error. Any ideas?
![Screenshot 2024-03-12 091122](https://github.com/agersant/polaris/assets/156863147/aaa81e87-c573-4fd8-bf2d-c50890ea4a2a)

**Comment by agersant** [comment, marked as answer]
Thank you for the report. I just release version 0.14.2 which should address this.
